funfact/algorithm.py

```diff
@@ -179,6 +179,6 @@ def loss_and_penalty(model, target, sum_vec=True):
             np.argsort(best_loss)[:returns]
         ]
     elif returns == 'all':
-        return best_fac
-    else:
-        raise RuntimeError(f'Invalid argument value for returns: {returns}')
+        return [
+            view(best_fac, tsrex, i, append) for i in np.argsort(best_loss)
```

Comment:
I agree that it's good for consistent output that you changed this. Also update the doc string about `returns` being `'all'` and probably the user guide: https://funfact.readthedocs.io/en/latest/pages/user-guide/factorize/#vectorization-for-multi-instance-learning